alf/algorithms/actor_critic_algorithm.py

```diff
@@ -16,7 +16,7 @@
 import gin.tf
 import tensorflow as tf
 
-from tf_agents.networks.network import Network, DistributionNetwork
+from alf.utils.networks import Network, DistributionNetwork
```

Comment:
Now from alf.networks import ...

Reply:
Changed

Reply:
@hnyu changes have done according to your comments. Do you have any more comments?